create_derivatives/helpers.py

```diff
@@ -1,7 +1,35 @@
+from os import listdir
+from os.path import isfile, join
```

Comment:
Let's use `pathlib` instead of `os` - https://docs.python.org/3.6/library/pathlib.html

You may want to coordinate with @bonniegee because she added this same helper.